### ISSUE TITLE
Calling craftItem from ME-Bridge (connected@AE2) returns Java.IllegalStateException

### ISSUE THREAD

**Issue by lephro** [Seniorendi/AdvancedPeripherals#188]
### Descripe

Calling craftItem from ME-Bridge (connected at a Applied Energistics 2 network) returns a Java.IllegalStateException 

(Latest AP + AE2 version)

### Steps to reproduce

```
bridge = peripheral.find("meBridge")
bridge.craftItem({ name="minecraft:gray_dye", count=1 })
```

### Multiplayer?

Yes

### Version

0.7.1r (Latest)

### Minecraft, Forge and maybe other related mods versions

36.2.2

### Screenshots or Videos

https://ibb.co/tXYdt5n

**Comment by SirEndii** [comment]
@SirEdvin this is your fault :P

I will fix that in the next update

**Comment by lephro** [comment]
@SirEdvin Shame!  :P :P :P :P

Your mod is awesome btw, guys! Thanks for doing this.

**Comment by SirEdvin** [comment]
Should be fixed in 0.7.2r
